ceph/ceph_admin/dashboard.py

```diff
@@ -10,6 +11,29 @@
 LOG = Log(__name__)
 
 
+def execute_commands(cls, commands):
+    for cmd in commands:
+        out, err = cls.shell(args=[cmd])
+        LOG.info("STDOUT:\n %s" % out)
+        LOG.error("STDERR:\n %s" % err)
+
+
+def validate_url(url):
+    """Method to validate provided API URL.
+    The HTTP 200 OK success status response code indicates that the request has succeeded.
+
+    Args:
+        url: URL of any API.
+    """
+    url = url.rstrip()
+    resp = requests.get(url, verify=False)
```

Comment:
@sunilkumarn417 yes to avoid SSL certificate security exception added verify set to false 
So it just checks the URL request is valid.

Reply:
@sunilkumarn417 Please review the changes all review comments are addressed.
Test results: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-3SMP0I

Reply:
@sunilkumarn417 some times due to rhos-d infra connection issue it takes more time to reach grafana even while manual login it takes more time hence this API validation fails is Ok for me.
also in one of the run, it's successful 
test results: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-3899VB